consul_exporter.go

```diff
@@ -86,6 +86,15 @@ func NewExporter(uri string) *Exporter {
 			[]string{"service", "node"},
 		),
 
+		nodeChecks: prometheus.NewGaugeVec(
+			prometheus.GaugeOpts{
+				Namespace: namespace,
+				Name:      "agent_check",
```

Comment:
The naming and help string on this one could still be improved - how about naming it `agent_check_passing` to make it sound like a boolean, and then also document that it can take the values of `0` or `1` in the docstring?